isodesign/process/calculation.py

```diff
@@ -339,49 +377,53 @@ def generate_mixes(self, tracers: dict):
+        logger.info("Isotopomers : %s %s\n", self.mix.names, self.mix.patterns)
+        logger.info("Isotopomers combinations : %s", self.mix.isotopomers_combination)
 
-    def generate_file(self, output_path:str):
+    def generate_linp_files(self, output_path: str):
         """
-        Generate linp files to a folder in the current directory depending
-        of all combination for one or two mixe(s). Those files are used 
-        for influx_si simulations.
-        Files containing tracers features including the combinations
-        for all tracer mixes.
+        Generate linp files to a folder in the current directory depending 
+        of all combinations for one or multiple isotopomers groups. 
+        Those files are used for influx_si simulations.
+        Files have a number as name and contained isotopomers 
+        features including the combinations for all isotopomer group.
+
+        A file is generated containing all the file number and combinations 
+        informations. 
         """
-
-        output_path = Path(output_path)
         
-        for combination in self.mix.mixes:
-            df = pd.DataFrame({'Id': None,
-                               'Comment': None,
-                               'Specie': [],
-                               'Isotopomer': [],
-                               'Value': []})
-
-            df["Specie"] = [tracer_names for tracer_names in self.mix.names]
-            df["Isotopomer"] = [tracer_isotopomer for tracer_isotopomer in self.mix.isotopomer]
-
-            if len(self.mix.tracers) > 1:
-                # Tuple contained tuples with all the combination reunited 
-                reunited_combination = ()
-                for pair in combination:
-                    reunited_combination += pair
-                df["Value"] = reunited_combination
-            else:
-                df["Value"] = combination
+        counter = 1
+
+        # Create the folder that stock linp files if is doesn't exist
+        if not os.path.exists(output_path):
+            os.makedirs(output_path)
+        self.path_linp_folder = os.path.abspath(output_path)
+
+        logger.info("Creation of the linp files...")
+
+        with open(os.path.join(str(output_path), 'files_combinations.txt'), 'w', encoding="utf-8") as f:
+            for pair in self.mix.isotopomers_combination["combinations"]:
+                df = pd.DataFrame({'Id': None,
+                                    'Comment': None,
+                                    'Specie': [],
+                                    'Isotopomer': [],
+                                    'Value': []})
+
+                df["Specie"] = list(self.mix.names)
+                df["Isotopomer"] = list(self.mix.patterns) 
+                df["Value"] = pair
+                df = df.loc[df["Value"] != 0]
+                
+                df.to_csv(os.path.join(str(output_path), f"{counter}.linp"), sep="\t")   
```

Comment:
Ici je ferais un log debug avec le dataframe, le numero d'itération et le chemin de sortie!